api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -43,23 +46,29 @@
 
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
-  public ResponseEntity<Void> updateUserDisabledStatus(UpdateUserDisabledRequest request) {
-    final DbUser targetDbUser = userDao.findUserByUsername(request.getEmail());
-    final Boolean previousDisabled = targetDbUser.getDisabled();
-    final DbUser updatedTargetUser =
-        userService.setDisabledStatus(targetDbUser.getUserId(), request.getDisabled());
-    auditAdminActions(request, previousDisabled, updatedTargetUser);
+  public ResponseEntity<Void> updateDisabledStatusForUsers(
+      UpdateDisabledStatusForUsersRequest request) {
+    final List<DbUser> dbUsers = userDao.findUsersByUsernameIn(request.getEmailList());
+    final List<DbUser> updatedDbUsers =
+        userService.setDisabledStatusForUsers(dbUsers, request.getDisabled());
+    final Map<Long, Boolean> updatedDbUsersByUserId =
+        updatedDbUsers.stream().collect(Collectors.toMap(DbUser::getUserId, DbUser::getDisabled));
+    dbUsers.forEach(
+        user -> {
+          auditAdminActions(
```

Comment:
Note to self: think about whether this should all be logged as one operation or many.

Have you verified in the local audit log that these are showing up?